WordPress/Classes/ViewRelated/Stats/SiteStatsTableViewCells.swift

```diff
@@ -420,7 +462,7 @@ struct PostStatsEmptyCellHeaderRow: ImmuTableRow {
             return
         }
 
-        cell.configure(withTitle: "", adjustHeightForPostStats: true)
+        cell.configure(statSection: .postStatsGraph)
```

Comment:
How come the empty cell header row is using the post stats graph section? (I may just be misunderstanding the code here)

Reply:
On the `Post Stats` view, the `StatsCellHeader` above the graph has a specific height. While I was refactoring the `configure` method, I realized I didn't explicitly need the `adjustHeightForPostStats` param, all I needed to do was tell it the section it was for. By sending `postStatsGraph`, `StatsCellHeader` sizes itself correctly because:
1. `StatsCellHeader` knows it's for the graph header.
2. `postStatsGraph` doesn't have a title.

I can add a comment in my next PR to clarify that.